.circleci/config.yml

```diff
@@ -239,6 +239,9 @@ jobs:
         type: string
       toxenv:
         type: string
+      shard:
```

Comment:
this is a dummy parameter to make circle config parser happy when expanding the matrices below.